src/domain/accounts/domain/src/services/authentication_provider.rs

```diff
@@ -8,17 +8,19 @@
 // by the Apache License, Version 2.0.
 
 use internal_error::InternalError;
-use opendatafabric::AccountName;
 use thiserror::Error;
 
-use super::{AccountInfo, InvalidCredentialsError, RejectedCredentialsError};
+use super::{InvalidCredentialsError, RejectedCredentialsError};
+use crate::Account;
 
 ///////////////////////////////////////////////////////////////////////////////
 
 #[async_trait::async_trait]
 pub trait AuthenticationProvider: Sync + Send {
     fn login_method(&self) -> &'static str;
 
+    fn get_predefined_accounts(&self) -> Vec<Account>;
```

Comment:
Should predefined accounts be a feature of auth service, not a provider? If we are defining account in config - we will never reach out to external provider about it, so what is the point of having "GH" or "LDAP" predefined account.

Reply:
Actually, most logically to me seems to make predef accounts a feature of Login+Password provider.

Reply:
On today the configration mechanism is CLI specific. Node has a different mechanism,

Reply:
Moved predefined accounts to service as suggested.